enterprise/backend/src/metabase_enterprise/serialization/v2/models.clj

```diff
@@ -33,3 +36,45 @@
    "Dimension"
    "ParameterCard"
    "TimelineEvent"])
+
+(def excluded-models
+  "List of models which are not going to be serialized ever."
+  ["Activity"
+   "ApplicationPermissionsRevision"
+   "AuditLog"
+   "BookmarkOrdering"
+   "CardBookmark"
+   "CollectionBookmark"
+   "CollectionPermissionGraphRevision"
+   "ConnectionImpersonation"
+   "DashboardBookmark"
+   "DashboardCardSeries"
+   "Database"
+   "Field"
```

Comment:
These should not be listed here - they're `data-model`s.

Reply:
eh and I thought I'd cleaned the list :\ fixed